app/models/course/assessment/question.rb

```diff
@@ -38,7 +40,7 @@ class Course::Assessment::Question < ApplicationRecord
   #
   # @return [Boolean] True if the question supports auto grading.
   def auto_gradable?
-    actable.present? && actable.self_respond_to?(:auto_gradable?) ? actable.auto_gradable? : false
+    (actable.present? && actable.self_respond_to?(:auto_gradable?)) ? actable.auto_gradable? : false
```

Comment:
This was an automatic change made by rubocop (Style/Ternary/Parentheses) that says parentheses should be used for ternary expressions with complex conditions

Reply:
I see.. so I guess because you modified this file then Hound sent you the Rubocop error for this style/ternary thing. Okay understand, it should be fine then